src/components/Description.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Description = ({ name, content, className }) => <div className={ className } id={ name }>
```

Comment:
add
(
 <div...
)

And we should add a linter rule for this.

Reply:
The rule was added! :straight_ruler: